src/Components/Notifications/__tests__/MarkAllAsReadPanel.jest.tsx

```diff
@@ -16,12 +16,12 @@ describe("MarkAllAsReadPanel", () => {
   it("should the single state", () => {
     render(<Notifications mode="page" unreadCounts={1} />)
 
-    expect(screen.getByText("1 new notification")).toBeInTheDocument()
+    expect(screen.getByText("New notifications")).toBeInTheDocument()
```

Comment:
I've decided not to alter tests a lot because I expect that this change is very short-lived and we are going to bring back counter soon, so I tried not to change a lot of things